### ISSUE TITLE
Use a cert from the machine store. Without a password.

### ISSUE THREAD

**Issue by DSBloom** [StefanKert/azuredevops-codesigning-task#17]
This tool seems to be designed to use a cert file with a password. Could a future version allow the use of a cert from the cert store? Specified by thumbprint? 

This is how we are currently signing code, via a PowerShell script. It would be nice to have it handled with a build task like this, though.

**Comment by DSBloom** [comment]
I actually wrote an extension that does this. Would you be interested in this functionality?

**Comment by StefanKert** [comment]
Hi @nadcraker,

I´d greatly appreciate it if you could send me some information on this. Is your extension on GitHub?

**Comment by DSBloom** [comment]
No, I'm afraid it's not. In the meantime I found this extension that seems to implement the functionality I needed. https://github.com/jabbera/vsts-authenticode

It allows you to use either a .pfx file or a cert from the user or machine store by specifying a thumbprint. Which is exactly what we are doing in my organization.

**Comment by StefanKert** [comment]
Hi @nadcraker,

very sorry for the long delay. I´ll double check that and will try to get it in asap :)
